processor/src/main/kotlin/permissions/dispatcher/processor/impl/kotlin/KotlinBaseProcessorUnit.kt

```diff
@@ -109,6 +109,7 @@ abstract class KotlinBaseProcessorUnit : KtProcessorUnit {
 
     private fun createWithPermissionCheckFun(rpe: RuntimePermissionsElement, method: ExecutableElement): FunSpec {
         val builder = FunSpec.builder(withPermissionCheckMethodName(method))
+                .addOriginatingElement(rpe.element)
```

Comment:
Do we need to add an original element for every spec builder?👀

Reply:
It might work if we set an origin element for just one spec that's guaranteed to be present in the file. But I think it's better to be safe and explicit and set origin element for every top-level spec we produce.
KotlinPoet 1.3.0+ deduplicates origin elements before passing them to Filer so kapt will see only one origin element.